src/pages/index.js

```diff
@@ -93,12 +119,10 @@ function handlePlaceCreateForm(data) {
     link: data.link,
   };
   placeCreateFormValidator.disableSubmit();
```

Comment:
`placeCreateFormValidator.disableSubmit();` should be in line 123 in `then`


You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.